MapboxDirections/MBRouteStep.swift

```diff
@@ -527,7 +527,7 @@ public class RouteStep: NSObject, NSSecureCoding {
         destinations = json["destinations"] as? String
         
         let maneuver = json["maneuver"] as! JSONDictionary
-        instructions = maneuver["instruction"] as! String
+        instructions = maneuver["instruction"] as? String
```

Comment:
Does this really have to be optional? That makes the library less convenient for the most common use case, which involves the Mapbox Directions API. Can non-Mapbox uses make do with an empty string or a very basic generated sentence?